Services/GEV/ReportsRepository/classes/class.ilBaseReportPlugin.php

```diff
@@ -0,0 +1,8 @@
+<?php
+include_once 'Services/Repository/classes/class.ilRepositoryObjectPlugin.php';
+
+class ilBaseReportPlugin extends ilRepositoryObjectPlugin {
+
+	abstract public function getPluginName();
```

Comment:
Vielleicht auf den Namen des Reports ummünzen und in dem Report lieber "getReportName" implementieren?